examples/bart/README.md

```diff
@@ -10,10 +10,11 @@ BART is sequence-to-sequence model trained with denoising as pretraining objecti
 
 - CNN daily mail validation data, NVIDIA-V100-16GB
 
-  |     BatchSize    |       32      |        64       |      128       |
-  |:----------------:|:-------------:|:---------------:|:--------------:|
-  | fairseq-0.9.0    | 2.7 samples/s |       OOM       |      OOM       |
-  | above + fastseq  | 9.0 samples/s | 13.0 samples/s  | 15.0 samples/s |
+  |     BatchSize    |       32      |        64       |      128       |      256*      |
+  |:----------------:|:-------------:|:---------------:|:--------------:|:--------------:|
+  | fairseq-0.9.0    | 2.4 samples/s |       OOM       |      OOM       |      OOM       |
+  | above + fastseq  | 8.1 samples/s | 13.3 samples/s  | 18.4 samples/s | 19.7 samples/s |
```

Comment:
bs32 speed decreased?

Reply:
Maybe because the average length of 1024 sample subset is shorter, or it made by duplicate 128 subset.